src/LogicApp/LogicApp/Cmdlets/IntegrationAccount/UpdateAzureIntegrationAccountCommand.cs

```diff
@@ -51,6 +51,7 @@ public class UpdateAzureIntegrationAccountCommand : LogicAppBaseCmdlet
 
         [Parameter(Mandatory = false, HelpMessage = "The integration account SKU.",
             ValueFromPipelineByPropertyName = true)]
+        [ValidateSet("Free", "Basic", "Standard", IgnoreCase = true)]
```

Comment:
same comment here about errors/breaking changes

Reply:
My response is the same as the other comment :)